docs/user_guide/logging.rst

```diff
@@ -0,0 +1,57 @@
+Logging
+=======
+
+Since version 0.18.0 :program:`rebase-helper` uses standard module-level loggers.
+:program:`rebase-helper` uses :class:`rebasehelper.logger.CustomLogger` logger class which provides extra
+logging levels.
+
+Default console handlers can be setup by calling :meth:`rebasehelper.logger.create_stream_handlers` and default file
+handlers by calling :meth:`rebasehelper.logger.create_file_handlers`.
+
+There are also 3 loggers with a special purpose:
+
+=============================== =====================================================================
+Logger name                     Purpose
+=============================== =====================================================================
+:samp:`rebasehelper.traceback`  traceback generated by unhandled exceptions
+:samp:`rebasehelper.summary`    messages forming the rebase summary
+:samp:`rebasehelper.report`     messages forming the textual rebase report (used by text output tool)
+=============================== =====================================================================
```

Comment:
It would make more sense if the table was just after the first sentence, because it also describes loggers.